taccsite_cms/templates/djangocms_blog/includes/blog_meta.html

```diff
@@ -61,21 +61,7 @@
 <ul class="post-detail categories">{% spaceless %}
 {# /TACC #}
 {# /TACC #}
-    {% if post.categories.exists %}
-        {% for category in post.categories.all %}
-            {% if category.slug %}
-                {# TACC (fix bug nephila/djangocms-blog#711): #}
-                {# TACC (remove hard-coded comma so CSS handles separator): #}
-                {# TACC (rename `blog-categories-…` to `blog-category-…`): #}
-                <li class="category_{{ forloop.counter }}"><a href="{% url 'djangocms_blog:posts-category' category=category.slug %}" class="blog-category-{{ category.slug }}">{{ category.name }}</a></li>
-                {# /TACC #}
-                {# /TACC #}
-                {# /TACC #}
-            {% endif %}
-        {% endfor %}
-    {% else %}
-      <li class="category_none">{% trans "Uncategorized" %}</li>
-    {% endif %}
+{% include './blog_cats_organized.html' %}
```

Comment:
I promise to stop commenting on differences from original template… after [TUP-656](https://tacc-main.atlassian.net/browse/TUP-656) or after upgrading to [djangocms-blog v2](https://github.com/nephila/djangocms-blog/releases).

```suggestion
    {# TACC (nephila/djangocms-blog#730, no commas, rename class, order cats): #}
    {% include './blog_cats_organized.html' %}
    {# /TACC #}
```